gatsby-config.js

```diff
@@ -4,6 +4,8 @@ module.exports = {
     description: `ReactJS Enthusiasts in the Philippines`,
     author: `ReactJS PH`,
     email: `reactjs.ph@yahoo.com`,
+    cover: `/images/cover.jpg`,
```

Comment:
I'm not sure if this would work, the only way to test this is to deploy the site.

Maybe move the `cover.jpg` to `static` folder. [About static folder of Gatsby](https://www.gatsbyjs.org/docs/static-folder/)

Reply:
Push some changes moving the cover to the static folder.